packages/angular_devkit/build_angular/src/builders/dev-server/load-proxy-config.ts

```diff
@@ -6,10 +6,12 @@
  * found in the LICENSE file at https://angular.io/license
  */
 
+import isGlob from 'is-glob';
```

Comment:
Instead of adding an additional dependency, the `glob` library `hasMagic` method can be used to check is the input is glob.